reactor-stream/src/main/java/reactor/rx/action/transformation/GroupByAction.java

```diff
@@ -160,9 +160,7 @@ protected void doComplete() {
 			stream.onComplete();
 		}
 
-		if (groupByMap.isEmpty()) {
-			broadcastComplete();
-		}
+		super.doComplete();
```

Comment:
I'm not sure the previous intention, but if there are no more incoming values, it follows that that there will be no more grouped streams created by this action.